src/sql/base/browser/dom.ts

```diff
@@ -22,21 +21,22 @@ export function convertSize(size: number | string | undefined, defaultValue?: st
 		return defaultValue;
 	}
 	let convertedSize: string = size ? size.toString() : defaultValue;
-	if (!endsWith(convertedSize.toLowerCase(), 'px') && !endsWith(convertedSize.toLowerCase(), '%')) {
+	convertedSize = convertedSize.toLowerCase();
```

Comment:
not related to the issue, I was looking at the implementation here and saw the obsolete functions being used here. so I replaced the obsolete endsWith function with String.endsWith function